README.md

```diff
@@ -12,16 +12,11 @@ $ npm install --save sentry-winston
 
 ```js
 var WinstonSentryTransport = require('sentry-winston').default;
-this.logger.add(WinstonSentryTransport, {sentry:sentryInstance}, false);
+this.logger.add(new WinstonSentryTransport({sentry}));
 ```
 ```js
 this.logger.log(level, "This is a message");
-this.logger.log(level, "This is an error", {error:new Error("I'm an error")});
```

Comment:
It seems like we're still supporting `error` and `info` fields based on https://github.com/pramp/sentry-winston/pull/1/files#diff-7617ad71241036df0f3d77ba7fa3b17eR49

Should this be changed?

Reply:
@i-van ?

Reply:
updated README
they (winston) changed API